<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>ISD_QT2</class>
	<widget class="QMainWindow" name="ISD_QT2">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>1280</width>
				<height>800</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>Информационная система деканата</string>
		</property>
		<widget class="QWidget" name="centralwidget">
			<layout class="QVBoxLayout" name="verticalLayout_root">
				<item>
					<widget class="QTabWidget" name="tabWidget">
						<property name="currentIndex">
							<number>0</number>
						</property>

						<!-- Students tab -->
						<widget class="QWidget" name="tabStudents">
							<attribute name="title">
								<string>Студенты</string>
							</attribute>
							<layout class="QVBoxLayout" name="verticalLayoutStudents">
								<item>
									<layout class="QHBoxLayout" name="layoutSearchStudents">
										<item>
											<widget class="QLineEdit" name="editSearchStudent">
												<property name="placeholderText">
													<string>Поиск по ФИО…</string>
												</property>
											</widget>
										</item>
										<item>
											<widget class="QPushButton" name="btnSearchStudent">
												<property name="text">
													<string>Поиск</string>
												</property>
											</widget>
										</item>
										<item>
											<spacer name="spacerStudents">
												<property name="orientation">
													<enum>Qt::Horizontal</enum>
												</property>
												<property name="sizeHint">
													<size>
														<width>40</width>
														<height>20</height>
													</size>
												</property>
											</spacer>
										</item>
										<item>
											<widget class="QPushButton" name="btnImportCSV">
												<property name="text">
													<string>Импорт CSV</string>
												</property>
											</widget>
										</item>
									</layout>
								</item>
								<item>
									<widget class="QTableWidget" name="tblStudents">
										<property name="editTriggers">
											<set>QAbstractItemView::NoEditTriggers</set>
										</property>
										<property name="selectionMode">
											<enum>QAbstractItemView::SingleSelection</enum>
										</property>
										<property name="selectionBehavior">
											<enum>QAbstractItemView::SelectRows</enum>
										</property>
										<property name="alternatingRowColors">
											<bool>true</bool>
										</property>
									</widget>
								</item>
								<item>
									<layout class="QHBoxLayout" name="horizontalLayoutStudents">
										<item>
											<widget class="QPushButton" name="btnAddStudent">
												<property name="text">
													<string>Добавить студента</string>
												</property>
											</widget>
										</item>
									</layout>
								</item>
							</layout>
						</widget>

						<!-- Groups tab -->
						<widget class="QWidget" name="tabGroups">
							<attribute name="title">
								<string>Группы</string>
							</attribute>
							<layout class="QVBoxLayout" name="verticalLayoutGroups">
								<item>
									<layout class="QHBoxLayout" name="layoutSearchGroups">
										<item>
											<widget class="QLineEdit" name="editSearchGroup">
												<property name="placeholderText">
													<string>Поиск по номеру группы…</string>
												</property>
											</widget>
										</item>
										<item>
											<widget class="QPushButton" name="btnSearchGroup">
												<property name="text">
													<string>Поиск</string>
												</property>
											</widget>
										</item>
									</layout>
								</item>
								<item>
									<widget class="QTableWidget" name="tblGroups"/>
								</item>
								<item>
									<layout class="QHBoxLayout" name="horizontalLayoutGroups">
										<item>
											<widget class="QPushButton" name="btnAddGroup">
												<property name="text">
													<string>Добавить группу</string>
												</property>
											</widget>
										</item>
									</layout>
								</item>
							</layout>
						</widget>

						<!-- Teachers tab -->
						<widget class="QWidget" name="tabTeachers">
							<attribute name="title">
								<string>Преподаватели</string>
							</attribute>
							<layout class="QVBoxLayout" name="verticalLayoutTeachers">
								<item>
									<layout class="QHBoxLayout" name="layoutSearchTeachers">
										<item>
											<widget class="QLineEdit" name="editSearchTeacher">
												<property name="placeholderText">
													<string>Поиск по ФИО…</string>
												</property>
											</widget>
										</item>
										<item>
											<widget class="QPushButton" name="btnSearchTeacher">
												<property name="text">
													<string>Поиск</string>
												</property>
											</widget>
										</item>
									</layout>
								</item>
								<item>
									<widget class="QTableWidget" name="tblTeachers"/>
								</item>
								<item>
									<layout class="QHBoxLayout" name="horizontalLayoutTeachers">
										<item>
											<widget class="QPushButton" name="btnAddTeacher">
												<property name="text">
													<string>Добавить преподавателя</string>
												</property>
											</widget>
										</item>
									</layout>
								</item>
							</layout>
						</widget>

						<!-- Specialties tab -->
						<widget class="QWidget" name="tabSpecialties">
							<attribute name="title">
								<string>Специальности</string>
							</attribute>
							<layout class="QVBoxLayout" name="verticalLayoutSpecialties">
								<item>
									<widget class="QTableWidget" name="tblSpecialties"/>
								</item>
								<item>
									<layout class="QHBoxLayout" name="horizontalLayoutSpecialties">
										<item>
											<widget class="QPushButton" name="btnAddSpecialty">
												<property name="text">
													<string>Добавить специальность</string>
												</property>
											</widget>
										</item>
									</layout>
								</item>
							</layout>
						</widget>

						<!-- Subjects tab -->
						<widget class="QWidget" name="tabSubjects">
							<attribute name="title">
								<string>Предметы</string>
							</attribute>
							<layout class="QVBoxLayout" name="verticalLayoutSubjects">
								<item>
									<widget class="QTableWidget" name="tblSubjects"/>
								</item>
								<item>
									<layout class="QHBoxLayout" name="horizontalLayoutSubjects">
										<item>
											<widget class="QPushButton" name="btnAddSubjectToSpec">
												<property name="text">
													<string>Добавить предмет</string>
												</property>
											</widget>
										</item>
									</layout>
								</item>
							</layout>
						</widget>

					</widget>
				</item>
			</layout>
		</widget>
		<widget class="QMenuBar" name="menubar">
			<property name="geometry">
				<rect>
					<x>0</x>
					<y>0</y>
					<width>1280</width>
					<height>21</height>
				</rect>
			</property>
		</widget>
		<widget class="QStatusBar" name="statusbar"/>
	</widget>
	<resources/>
	<connections/>
</ui>
